corehq/apps/users/views/mobile/users.py

```diff
@@ -844,7 +855,7 @@ def _build_commcare_user(self):
 
         return CommCareUser.create(
             self.domain,
-            format_username(username, self.domain),
+            username,
```

Comment:
Can you clarify what this change was for? Looks like it could have far-reaching side effects.